sqlglot/expressions.py

```diff
@@ -6273,19 +6273,15 @@ class ToDays(Func):
 
 
 class WeekOfYear(Func):
-    _sql_names = ["WEEK_OF_YEAR", "WEEKOFYEAR"]
-
-
-class WeekIso(Func):
-    _sql_names = ["WEEK_ISO", "WEEKISO"]
+    _sql_names = ["WEEK_OF_YEAR", "WEEKOFYEAR", "WEEKISO"]
 
 
 class YearOfWeek(Func):
-    _sql_names = ["YEAR_OF_WEEK", "YEAROFWEEK"]
+    pass
```

Comment:
I actually think the previous `_sql_names` were correct, for both this and `YearOfWeekIso`. This is one case where they contain "obvious" aliases of the expression being represented.